addon/components/materialize-tabs.js

```diff
@@ -0,0 +1,87 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-tabs';
+
+export default Ember.Component.extend({
+  layout: layout,
+  tagName: 'ul',
+  tabWidth: 2,
```

Comment:
Width of the contained tabs, in columns (can be overridden on a per-tab basis, optionally)